plugins/destination/clickhouse/queries/testdata/create_table.sql

```diff
@@ -1,9 +1,9 @@
 CREATE TABLE `table_name` (
-  `_cq_id` UUID,
-  `_cq_parent_id` Nullable(UUID),
-  `_cq_source_name` Nullable(String),
-  `_cq_sync_time` Nullable(DateTime64(9)),
-  `extra_col` Float64,
-  `extra_inet_col` Nullable(String),
-  `extra_inet_arr_col` Array(Nullable(String))
-) ENGINE = MergeTree ORDER BY (`_cq_id`)
\ No newline at end of file
+`_cq_id` UUID,
```

Comment:
could we leave the 2 spaces, though?
Less diff + easier query debug for those who look at the sdk logs

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/8255/commits/6764c84e1bf8c5bb15145c7509cc4c8f7303f098